server/server.go

```diff
@@ -118,6 +118,9 @@ type Server struct {
 	stats *statistics
 	// GetCertificate is used for loading certificates.
 	GetCertificate GetCertificate
+	// Sealer is called for Seal and Unseal operations.
+	// Sealer can return a gokeyless.Error to send a custom error code.
+	Sealer func(*gokeyless.Operation) ([]byte, error)
```

Comment:
I understand, my preference is that we have a `Sealer` interface, has `Seal` and `Unseal` functions, and in the switch block below, we actually match Seal with Opseal, Unseal with Opunseal. My feeling is that that is easier to make changes later on, and less error prone